airbyte-cdk/python/airbyte_cdk/sql/_processors/motherduck.py

```diff
@@ -54,7 +54,7 @@ def get_sql_alchemy_url(self) -> SecretString:
 
         return SecretString(
             f"duckdb:///md:{self.database}?motherduck_token={self.api_key}"
-            f"&custom_user_agent=={self.custom_user_agent}"
+            f"&custom_user_agent={self.custom_user_agent}"
```

Comment:
👍

Reply:
This won't get included in our release because it will need a separate CDK publish. That said, it should still be fine to include here - I'll just need to publish separately after the connector is released.